backend/internal/github/github.go

```diff
@@ -46,6 +46,8 @@ type GitHubUserClient interface { // All methods in the OAUTH client
 
 	// Callback for the OAUTH flow
 	GitHubCallback(code string, clientCfg config.GitHubUserClient) (string, error)
+
+	ForkRepository(ctx context.Context, org, owner, repo, destName string) error
```

Comment:
What was the decision for doing this? While we want to fork from users in your approach, are there instances in the future we would want to preserve the ability to do this as an app?

Reply:
There's different functionality depending on if you do it from the app vs user. Has to do with destination and permissions of the resulting repo. 

Potentially yes, but if so it should probably be reimplemented under the app client so that devs understand behavior difference